plugins/modules/remote_management/redfish/wdc_redfish_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: wdc_redfish_info
+short_description: Manages WDC UltraStar Data102 Out-Of-Band controllers using Redfish APIs
```

Comment:
You also need a `version_added` here. (Basically all comments on the first module also apply to the second.)

Reply:
Fixed [7d838381d41f0620ea3b7406cda0762e971e8143](https://github.com/ansible-collections/community.general/pull/4885/commits/7d838381d41f0620ea3b7406cda0762e971e8143)